.github/workflows/benchmark.yml

```diff
@@ -17,66 +17,42 @@ permissions:
-      arangodb:
-        image: 'arangodb:latest'
-        env:
-          ARANGO_NO_AUTH: 1
-        ports:
-          - '8529:8529'
-      dynamodb:
-        image: 'amazon/dynamodb-local:latest'
-        ports:
-          - '8000:8000'
-      memcached:
-        image: 'memcached:latest'
-        ports:
-          - '11211:11211'
-      mongo:
-        image: 'mongo:latest'
-        ports:
-          - '27017:27017'
-      mssql:
-        image: 'mcmoe/mssqldocker:latest'
-        ports:
-          - '1433:1433'
-        env:
-          ACCEPT_EULA: Y
-          SA_PASSWORD: MsSql!1234
-          MSSQL_DB: master
-          MSSQL_USER: sa
-          MSSQL_PASSWORD: MsSql!1234
-        options: >-
-          --health-cmd "/opt/mssql-tools/bin/sqlcmd -U sa -P $SA_PASSWORD -Q 'select 1' -b -o /dev/null"
-          --health-interval 1s
-          --health-timeout 30s
-          --health-start-period 10s
-          --health-retries 20
-      mysql:
-        image: 'mysql:latest'
-        env:
-          MYSQL_DATABASE: fiber
-          MYSQL_USER: username
-          MYSQL_PASSWORD: password
-          MYSQL_ROOT_PASSWORD: password
-        ports:
-          - '3306:3306'
-        options: >-
-          --health-cmd "mysqladmin ping" --health-interval 10s --health-timeout
-          5s --health-retries 5
-      postgres:
-        image: 'postgres:latest'
-        ports:
-          - '5432:5432'
+    outputs:
+      modules: ${{ steps.set-modified-modules.outputs.modules }}
+      modules_count: ${{ steps.set-modified-modules-count.outputs.modules_count }}
+    steps:
+      - name: Check out code into the Go module directory
+        uses: actions/checkout@692973e3d937129bcbf40652eb9f2f61becf3332 # v4
+
+      - id: changed-files
+        name: Get changed files
+        uses: tj-actions/changed-files@4edd678ac3f81e2dc578756871e4d00c19191daf # v45.0.4
+
+      - id: set-modified-modules
+        name: Set all modified modules
         env:
-          POSTGRES_DB: fiber
-          POSTGRES_USER: username
-          POSTGRES_PASSWORD: "pass#w%rd"
-        options: >-
-          --health-cmd pg_isready --health-interval 10s --health-timeout 5s
-          --health-retries 5
+          ALL_CHANGED_FILES: "${{ steps.changed-files.outputs.all_changed_files }}"
+        run: echo "modules=$(./.github/scripts/changed-modules.sh)" >> $GITHUB_OUTPUT
+
+      - id: set-modified-modules-count
+        name: Set all modified modules count
+        run: echo "modules_count=$(echo ${{ toJSON(steps.set-modified-modules.outputs.modules) }} | jq '. | length')" >> $GITHUB_OUTPUT
+
+      - name: Print out the modules to be used
+        run: |
+          echo "${{ steps.set-modified-modules-count.outputs.modules_count }} modules in the build"
+          echo "${{ steps.set-modified-modules.outputs.modules }}"
+
+  Compare:
+    needs:
+      - detect-modules
+    strategy:
+      matrix:
+        module: ${{ fromJSON(needs.detect-modules.outputs.modules) }}
```

Comment:
I think minio for s3 benchmarks is missing here too

Reply:
I think the error is not because of not having a dedicated minio service in the GH workflow, but this https://github.com/gofiber/storage/pull/1536#discussion_r1913188690